src/main/scala/org/camunda/feel/api/FeelEngineApi.scala

```diff
@@ -197,7 +197,7 @@ class FeelEngineApi(engine: FeelEngine) {
     *   the result of the evaluation as [[EvaluationResult]]
     */
   def evaluateExpression(expression: String, context: Context): EvaluationResult =
-    engine.parseExpression(expression = expression.translateEscapes()) match {
+    engine.parseExpression(expression = StringContext.processEscapes(expression)) match {
```

Comment:
If I move the escaping function in the suggested method I encounter 2 errors

- [escaping \s](https://github.com/camunda/feel-scala/blob/2617de8a37bbff7d5d3d090ec654f0d2750c9546/src/test/scala/org/camunda/feel/impl/builtin/BuiltinStringFunctionsTest.scala#L126-L130), because the scala function does not support this sequence
- [escaping \d](https://github.com/camunda/feel-scala/blob/2617de8a37bbff7d5d3d090ec654f0d2750c9546/src/test/scala/org/camunda/feel/impl/builtin/BuiltinStringFunctionsTest.scala#L86-L88), because it is not an escapable sequence

The solution could be create a custom function to handle also those cases, wdyt ? @saig0

Reply:
> If I move the escaping function in the suggested method I encounter 2 errors

Yes. The tests fail with the following error. 

```
invalid escape '\s' not one of [\b, \t, \n, \f, \r, \\, \", \', \uxxxx] at index 20 in " split("John Doe", "\s") ". Use \\ for literal \.scala.StringContext$InvalidEscapeException: invalid escape '\s' not one of [\b, \t, \n, \f, \r, \\, \", \', \uxxxx] at index 20 in " split("John Doe", "\s") ". Use \\ for literal \.
	at scala.StringContext$.loop$2(StringContext.scala:396)
	at scala.StringContext$.replace(StringContext.scala:408)
	at scala.StringContext$.processEscapes(StringContext.scala:366)
```

The failure didn't occur before because `StringContext.processEscapes()` was not invoked in these tests. :laughing: 

> The solution could be create a custom function to handle also those cases

How would this look like? Please describe in more detail. 

One option could be to process the escapes after the evaluation before returning the result. 

Do you know why and where `\n` is escaped in the first place?

Reply:
My idea is to create a function similar of the `StringContext.processEscapes()` scala function. But with the addition of special cases, like `\s` or `\d`

> Do you know why and where `\n` is escaped in the first place?

The above test case suggest that we didn't escape characters, but we consider those characters as string, am I right ?

Reply:
> My idea is to create a function similar of the StringContext.processEscapes() scala function. But with the addition of special cases, like \s or \d

Okay. We can try that. :+1: 

> he above test case suggest that we didn't escape characters, but we consider those characters as string, am I right ?

As a user, you want to write an expression `split("John Doe", "\s")` that contains `\s`. We need to make sure that this works with the built-in functions.